spring-boot-project/spring-boot-docs/src/docs/asciidoc/io/hazelcast.adoc

```diff
@@ -8,7 +8,7 @@ Spring Boot first attempts to create a client by checking the following configur
 * A configuration file defined by the configprop:spring.hazelcast.config[] property.
 * The presence of the `hazelcast.client.config` system property.
 * A `hazelcast-client.xml` in the working directory or at the root of the classpath.
-* A `hazelcast-client.yaml` (or `hazelcast-client.yml`) in the working directory or at the root of the classpath.
+* A `hazelcast-client.yaml` in the working directory or at the root of the classpath.
```

Comment:
I don't think this change is necessary. Hazelcast supports both and the current docs indicate that `.yaml` is preferred.